employee-management-client/src/app/app.module.ts

```diff
@@ -1,18 +1,45 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { ClientsComponent } from './clients/clients.component';
+import { ClientComponent } from './clients/client/client.component';
+import { ClientListComponent } from './clients/client-list/client-list.component';
+import { ClientService } from './shared/client.service';
+import { CommonModule } from '@angular/common';
+import { MaterialModule } from './material/material.module';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import { SkillsComponent } from './skills/skills.component';
+import { SkillComponent } from './skills/skill/skill.component';
+import { SkillListComponent } from './skills/skill-list/skill-list.component';
+import { SkillService } from './shared/skill.service';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    ClientsComponent,
+    ClientComponent,
```

Comment:
Please rename this component as client-detail or client-info because it is ambiguous as clients component